libraries/common/array.effekt

```diff
@@ -128,24 +157,11 @@ def build[T](size: Int) { index: Int => T }: Array[T] = {
 
 // Utility functions:
 
-/**
- * Converts a List `list` to an Array
- */
-def fromList[T](list: List[T]): Array[T] = {
-  val listSize = list.size();
-  val arr = unsafeFresh(listSize);
-
-  foreachIndex(list) { (i, head) =>
-    unsafeSet(arr, i, head)
-  }
-  return arr;
-}
-
 def toList[T](arr: Array[T]): List[T] = {
   var i = arr.size - 1;
   var l = Nil[T]()
   while (i >= 0) {
-    l = Cons(unsafeGet(arr, i), l)
+    l = Cons(arr.unsafeGet(i), l)
```

Comment:
If I imagine writing `unsafe::get(arr, i)` or `arr.unsafe::get(i)` I am starting to thing that `unsafeGet` is nicer.

Reply:
Yeah, same, I originally had `arr.unsafe::get(i)`, but I didn't like the aesthetics of it.